SIS_import/rclone_setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash 
+
+# rclone --config ./rclone.conf config show
+rclone --config /app/config/secrets/rclone.conf lsd ~/
+
+rclone -vv --config /app/config/secrets/rclone.conf ls sftp:/OUTBOUND
+
+# SIS upload
+## 1.copy SIS zip file from the SFTP server
+rclone -vv --config /app/config/secrets/rclone.conf --exclude '.*' copy sftp:/OUTBOUND/ /usr/src/app/data
+
+
+## 2. start with the sis_upload.rb
+ruby /usr/src/app/sis_upload.rb
+
+if [ $? -eq 0 ] 
```

Comment:
We might want to capture the result of this return and ultimately return it at the end of the script. But probably Jenkins could also just be able to look for error in the logs. 

Otherwise the result of the sis_set_url.rb will be the return code of this script.